tests/drivers/build_all/dts_fixup.h

```diff
@@ -297,6 +297,99 @@
 #define DT_TI_TMP116_BUS_I2C 1
 #endif
 
+#ifndef DT_INST_0_HOPERF_TH02_LABEL
```

Comment:
Does it still make sense to keep adding those? It’s just for the non-DTS enabled platforms? Couldn’t the dt scripts generate them based on the overall DT files?

Reply:
It does for now, its the only way we have to ensure a build test for any driver that doesn't have an explicit sample.

The other option is to create a board overlay dts for this test that has all the sensors in it and that we'd keep adding as we have new sensor drivers.

Reply:
I do like that idea as it gives an example of what the .dts node should look like to someone wanting to add one of the sensors to their boards.